test/test_punktgruppe.py

```diff
@@ -0,0 +1,76 @@
+import fire
+from fire.api import FireDb
+from fire.api.model import (
+    Punkt,
+    PunktGruppe,
+    EventType,
+)
+
+
+def test_hent_punktgruppe(firedb):
+    """Test at vi kan læse punktgruppe fra databasen"""
+    punktgruppe = firedb.hent_punktgruppe("Aarhus Nivellementstest")
+
+    assert punktgruppe.jessenpunkt.ident == "RDIO"
+    assert punktgruppe.navn == "Aarhus Nivellementstest"
+
+    assert len(punktgruppe.punkter) == 3
+    for p in punktgruppe.punkter:
+        assert p.ident in ("K-63-19113", "K-63-09933", "K-63-09116")
+
+
+def test_hent_fra_punkt(firedb):
+    """Test at en punktgruppe kan findes via et punkt"""
+    punkt = firedb.hent_punkt("RDIO")
+    punktgruppe = punkt.punktgrupper[0]
+    assert isinstance(punktgruppe, PunktGruppe)
+
+
+def test_opret_punktgruppe(firedb, sagsevent, punktfabrik):
+    """Test at en punktgruppe kan oprettes"""
+
+    punkter = [punktfabrik() for _ in range(3)]
+
+    punktgruppe = PunktGruppe(
+        navn=f"Test-{fire.uuid()[0:9]}",
+        punkter=punkter,
+        jessenpunkt=punktfabrik(),
+    )
+
+    sagsevent.eventtype = EventType.PUNKTGRUPPE_MODIFICERET
+    sagsevent.punktgrupper = [punktgruppe]
+    firedb.indset_sagsevent(sagsevent)
```

Comment:
Jeg ville hive de oprettede data ud igen og se, at der er oprettet som forventet. Eller er metoden `indset_sagsevent` (eller andet i testkoden eller andre tests) en god nok garanti for dette?

Reply:
Det burde det jo være, men man kan aldrig vide sig for sikker. `indset_sagsevent` kalder et `commit()` på databasen så hvis ikke der kommer noget brok tilbage burde den være god nok. Der er jo en række regler sat op på databasen som også kan fejle her hvis ikke de er overholdt. Det burde også testes, jeg har indtil videre sprunget let og elegant over det.